doc/sphinx-guides/source/developers/big-data-support.rst

```diff
@@ -282,3 +282,25 @@ Available variables are:
 * ``minorVersion``
 * ``majorVersion``
 * ``releaseStatus``
+
```

Comment:
this doc is called big data support. Should we create something new for workflows generally and have this doc point to that?

Reply:
Sounds reasonable. I didn't want to start rearranging your docs, but I agree that, while there are big data uses for workflows, there's really nothing big-data specific about workflows or the included steps...

Reply:
Rearranging is fine. You can always ping @dlmurphy if you want some help sorting through the docs.